client/js/nonprofits/donate/wizard.js

```diff
@@ -154,6 +154,28 @@ const postDedication = (dedication, donor, donation) => {
   }).load)
 }
 
+const titleInfo = state => {
+  if (state.params$().title_image_url) {
+    return [
+      h('img', {
+        props: {
+          src: state.params$().title_image_url
+        , alt: state.params$().title_image_alt || app.campaign.tagline || app.nonprofit.tagline || ''
+        },
+      }),
+    ];
+  }
+
+  return [
+    h('h2', app.campaign.name || app.nonprofit.name)
+  , h('p', [
+      state.params$().designation && !state.params$().single_amount
+        ? headerDesignation(state)
+        : app.campaign.tagline || app.nonprofit.tagline || '',
```

Comment:
This does bring up an interesting question. Why do we handle this differently if it's a single amount?

Reply:
We'll answer this question some other time :smile: